### ISSUE TITLE
Verwijder overbodig geworden velden uit profiel bij veranderen status.

### ISSUE THREAD

**Issue by qurben** [csrdelft/csrdelft.nl#373]
De velden `kgb`, `novitiaat`, `vrienden`, `middelbareSchool`, `novietSoort`, `matrixPlek`, `startkamp`, `medisch` en `novitiaatBijz` verwijderen als de status van `S_NOVIET` naar `S_LID` gaat.

De velden `bankrekeningnr`, `o_adres`, `o_postcode`, `o_woonplaats`, `o_land`, `o_telefoon`, `studienr`, `eetwens`, `gebdatum` als de status van `S_LID` naar `S_OULID` gaat.

Misschien nog wat meer.

**Comment by mathcals** [comment]
Ja, dat kan sowieso. Die kan dan bij/verwijderdata.php runnen. Maar ik heb nog niet geïmplementeerd dat bij het wel of niet bewaren van een veld gekeken kan worden naar hoe lang iemand al oudlid is. Is niet heel moeilijk ofzo, maar nog niet gedaan.

**Comment by FrostMegaByte** [comment]
Deze issue heeft nog het label `prio:high`, hoe staat het hiermee? Een kleine update zou fijn zijn of kan deze issue eigenlijk al dicht?

**Comment by qurben** [comment]
De meuk verwijderen na 7 jaar moet nog gebouwd worden
